app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -151,10 +153,22 @@ class MessageActivity : AppCompatActivity() {
             }
         } else {
             Toast.makeText(this, "BLE is not supported", Toast.LENGTH_LONG).show()
+            send.isClickable = false
```

Comment:
That's not correct logic. You should disable the button instead. Making it non clickable will only annoy the users who won't even know why is it not registering their clicks. Also, as Mario said, there should be an error message saying that there is no device connected

Reply:
Yeah got that I would do the changes